pkg/kyverno/apply/command.go

```diff
@@ -102,7 +102,7 @@ func Command() *cobra.Command {
 
 					err = applyPolicyOnResource(policy, resource)
 					if err != nil {
-						return sanitizedError.New(fmt.Errorf("Issues applying policy %v on resource %v", policy.Name, resource.GetName()).Error())
+						return sanitizedError.NewWithError(fmt.Errorf("Issues applying policy %v on resource %v", policy.Name, resource.GetName()).Error(), err)
```

Comment:
Change to "failed to apply policy %v on resource %v"

Reply:
Ok sure.

Reply:
updated accordingly